docs/source/user-guide/configuration/settings.rst

```diff
@@ -97,6 +97,65 @@ handler called, "test-auth-handler" registered via the aforementioned plugin hoo
    schema must match exactly to the channel URL, so a pattern like ``*`` is not valid.
 
 
+``allowlist_channels`` and ``denylist_channels``: Allow or deny specific channels
+--------------------------------------------------------------------------------
+
+.. versionadded:: 24.9.0
+
+  The ``denylist_channels`` setting was introduced in conda 24.9.0 complementing the
+  existing ``allowlist_channels`` setting.
+
+With ``allowlist_channels`` and ``denylist_channels``, you can allow or deny specific channels
+from being used in conda operations. This is useful for restricting the channels that conda
+can access, especially in enterprise or multi-user environments.
+
+The denylist takes precedence over the allowlist. If a channel is in both lists, it is denied.
+
+**Examples:**
+
+An example which allows the ``defaults`` and ``conda-forge`` channels with the ``alllowlist_channels``
+setting is:
+
+.. code-block:: yaml
+
+  allowlist_channels:
+    - defaults
+    - conda-forge
+
+An example which denies the ``conda-forge`` channel with the ``denylist_channels`` setting is:
+
+.. code-block:: yaml
+
+  denylist_channels:
+    - conda-forge
+
+An example which explicitely allows the ``defaults`` channel but denies the ``conda-forge`` channel
+by using both the ``allowlist_channels`` and ``denylist_channels`` settings is:
+
+.. code-block:: yaml
+
+  allowlist_channels:
+    - defaults
+  denylist_channels:
+    - conda-forge
+
+An example to show that channels are automatically normalized based on their base URLs,
+so you can use either the full channel URL or just the base URL:
+
+.. code-block:: yaml
+
+  allowlist_channels:
+    - defaults
+  denylist_channels:
+    - https://conda.anaconda.org/conda-forge/linux-64
+
+An example that denies using the ``defaults`` channel:
+
+.. code-block:: yaml
+
+  denylist_channels:
+    - defaults
```

Comment:
Since `defaults` can be configured via `default_channels`, would it make more sense to have the `repo.anaconda.com` URLs listed here?

Reply:
Okay, let me actually mention that `default_channels` part and have both, that'll be important to clarify

Reply:
be9872aea27a0af2a2da43c69777db1796b30a6b